package.yaml

```diff
@@ -18,19 +15,6 @@ maintainer:       quasicomputational@gmail.com, Andreas Abel
 
 github: sol/doctest
 
-tested-with:
```

Comment:
You don't like this field?

Reply:
AFAIK, this field is only used by `haskell-ci`.  So unless I'm using `haskell-ci` I can't justify burning any cycles on keeping this up to date.